DirectXTex/DirectXTexCompress.cpp

```diff
@@ -492,95 +493,89 @@ namespace
 }
 
 //-------------------------------------------------------------------------------------
-namespace DirectX
+bool DirectX::Internal::IsAlphaAllOpaqueBC(_In_ const Image& cImage) noexcept
```

Comment:
No code changes in this function... just indented differently which shows up as a ton of diffs.